examples/e-commerce/src/app.css

```diff
@@ -281,3 +267,9 @@ mark {
 .toggle-refinement-checkbox:checked::before {
   content: 'Yes';
 }
+
+/* RatingMenu */
+
+.rating-menu-item:not(.rating-menu-item--selected) {
```

Comment:
It would be best to apply this style only when a RatingMenu item is refined but I didn't get to do it with only CSS.